classes/PodsInit.php

```diff
@@ -487,10 +487,8 @@ public function register_assets() {
 		 */
 		do_action( 'pods_before_enqueue_scripts', $suffix_min );
 
-		wp_register_script( 'pods-json', PODS_URL . 'ui/js/jquery.json.js', array( 'jquery' ), '2.3' );
-
 		if ( ! wp_script_is( 'jquery-qtip2', 'registered' ) ) {
-			wp_register_script( 'jquery-qtip2', PODS_URL . 'ui/js/jquery.qtip.min.js', array( 'jquery' ), '2.2' );
+			wp_register_script( 'jquery-qtip2', PODS_URL . "ui/js/qtip/jquery.qtip{$suffix_min}.js", array( 'jquery' ), '3.0.3', true );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: d5754925950536221768d0da749a06c5</sub>